rnaget-openapi.yaml

```diff
@@ -1240,6 +1253,10 @@ components:
         message:
           type: string
           description: Error message details
+
```

Comment:
I would remove this blank line

Reply:
blank line removed and references to version incremented to 1.1.0